metpy/cbook.py

```diff
@@ -39,4 +37,103 @@ def get_test_data(fname, as_file_obj=True):
     return path
 
 
-__all__ = ('get_test_data', 'is_string_like', 'iterable')
+class CacheData(object):
+    def __init__(self):
+        self.cache_home = self.get_cache_location()
+        self.cache_version = 1
```

Comment:
I think it makes sense to put the version as a class attribute, so put `cache_version = 1` above `__init__`.

Reply:
Done